src/main/java/de/tum/in/www1/artemis/service/TutorialGroupService.java

```diff
@@ -41,6 +50,10 @@ public void deregisterStudent(User student, TutorialGroup tutorialGroup, Tutoria
         tutorialGroupRegistrationRepository.delete(existingRegistration.get());
     }
 
+    public void deregisterMultipleStudentsFromAllTutorialGroupInCourse(Set<User> students, Course course, TutorialGroupRegistrationType registrationType) {
```

Comment:
```suggestion
    public void deregisterStudentsFromAllTutorialGroupInCourse(Set<User> students, Course course, TutorialGroupRegistrationType registrationType) {
```

Reply:
Renamed it in c7c6d5fb268c0385ea781798f6a07e8440f98294